libraries/lwc/meta/issues.json

```diff
@@ -0,0 +1,7 @@
+[
+  {
+    "link": "https://github.com/salesforce/lwc/issues/1904",
```

Comment:
This issue is closed now. Should it be https://github.com/salesforce/lwc/issues/3335 ?